Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -751,11 +743,7 @@ WarpXParticleContainer::PushX (int lev, amrex::Real dt)
             ParticleReal* AMREX_RESTRICT ux = attribs[PIdx::ux].dataPtr();
             ParticleReal* AMREX_RESTRICT uy = attribs[PIdx::uy].dataPtr();
             ParticleReal* AMREX_RESTRICT uz = attribs[PIdx::uz].dataPtr();
-#ifdef WARPX_DIM_RZ
-            auto& aos = pti.GetArrayOfStructs();
-            ParticleType* AMREX_RESTRICT const pstruct = aos().dataPtr();
-            ParticleReal* AMREX_RESTRICT theta = attribs[PIdx::theta].dataPtr();
-#endif
+
```

Comment:
Fixes the warnings:
`./Source/Particles/WarpXParticleContainer.cpp:748:48`: warning: unused variable ‘pstruct’ [-Wunused-variable]
             ParticleType* AMREX_RESTRICT const pstruct = aos().dataPtr();
`./Source/Particles/WarpXParticleContainer.cpp:749:42`: warning: unused variable ‘theta’ [-Wunused-variable]
             ParticleReal* AMREX_RESTRICT theta = attribs[PIdx::theta].dataPtr();